src/main/java/seedu/address/model/entity/worker/Designation.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.entity.worker;
+
+//@@author shaoyi
+/**
+ * Enumerates all possible designations of a worker.
+ */
+public enum Designation {
+
+    MANAGER, TECHNICIAN
```

Comment:
I think we should discuss this in the next meeting

Reply:
Agreed! Will add that to the agenda.